database/migrations/env.py

```diff
@@ -0,0 +1,67 @@
+from logging.config import fileConfig
+
+from sqlalchemy import engine_from_config
+from sqlalchemy import pool
+
+from alembic import context
+import os
+import sys
+
+sys.path.insert(
+    0, os.path.dirname(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
```

Comment:
maybe there's a better way to do this? seems like this will quietly break if directory structure is changed, which i can see being very difficult to debug